package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "videojs-vr",
-  "version": "1.10.1",
+  "version": "1.10.2",
```

Comment:
This gets updated as part of the release process and should be reverted.

Reply:
Doh! Didn't realise all good

Reply:
no worries!